emitterc.go

```diff
@@ -450,12 +451,16 @@ func yaml_emitter_emit_document_start(emitter *yaml_emitter_t, event *yaml_event
 			if !yaml_emitter_process_head_comment(emitter) {
 				return false
 			}
-			if !put_break(emitter) {
+			// Do not add newline if the document is empty
```

Comment:
This is just re-stating what we can read nicely in the line below.

Reply:
Removed in 193064d01c66d9643f3e6b9dd4142a2a83db62b4.